lib/msteams_hermes/message.rb

```diff
@@ -31,9 +56,20 @@ def deliver
         req = Net::HTTP::Post.new(uri)
         req.body = body_json
         req["Content-Type"] = "application/json"
-        http.request(req)
+
+        response = http.request(req)
+
+        if response.body != "1"
```

Comment:
That looks a bit fragile. How does the body look like in a "good" case.

Reply:
:D
![image](https://github.com/xing/msteams_hermes/assets/2405937/097781ff-3ac8-44af-a0f8-7d056a137fc0)
If that is "how to do it right", I do not comprehend. 
Alright then.

Reply:
Thank you for inquiring about it 🙏 
Since I did not asked myself enough questions about this piece of code 😞 

I know that it is brittle ..
But after this review I went to check and found: https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/connectors-using?tabs=cURL%2Ctext1#send-messages-using-curl-and-powershell
Which seems to say exactly "If the POST succeeds, you must see a simple 1 output by curl." 🤷‍♂️